cognite/neat/_session/_drop.py

```diff
@@ -23,6 +23,12 @@ def instances(self, type: str | list[str]) -> None:
 
         Args:
             type: The type of instances to drop.
+
+        Example:
```

Comment:
This is the other format type which will appear in a code block in the docs:

![Uploading Screenshot 2024-12-12 at 15.59.53.png…]()